build_node/build_node_builder.py

```diff
@@ -255,6 +255,9 @@ def __call_master(
             session_method = self.__session.get
         try:
             response = session_method(full_url, json=parameters)
+            # Special case when build was already done
+            if response.status_code == 409:
```

Comment:
`Requests` module have all status codes, so you can use `requests.codes.conflict` or `requests.codes.CONFLICT`

Reply:
Fixed